### PR TITLE
AB#10622 Approved amount is editable for approver only on any status

### PR DESCRIPTION
**Pull request by aurelio-aot**

### CODE REVIEW

**Comment by aurelio-aot** [comment]
If the user is non-approver, the approved amount is disabled, but the approved amount becomes zero if the non-approver edited other fields and save the form because the disabled field is sending null value in the backend.  That's why i need to enable the disabled approved amount just before sending the request to the backend.  Once the response comes back, the approved amount becomes disabled again.

**Review by AndreGAot — APPROVED** [review]
Ready for Dev

**Pull request merged**
